### ISSUE TITLE
[Feature Request] Pinned Stickers

### ISSUE THREAD

**Issue by karai17** [telegramdesktop/tdesktop#3333]
I love stickers. I love them a lot. But right now, stickers have a fundamental issue: There are SO MANY sticker packs, it can be difficult to keep track of all the stickers I use most.

Think of it like this, back in the 90s you'd buy a CD of some new band you heard on the radio only to find that of the 12 or so songs, you only really liked one or two. Over time, you grew a huge collexion of albums and when you wanted to listen to a particular song, you needed to sift through your albums and by the time you got there, the moment was over... If only there was a way to have all of your favourite songs without needing to worry about the cruft? Enter mp3s, where you had direct access to only the songs you wanted.

Back to Telegram, I think it would be super duper useful if we had the ability to pin individual stickers into our own groupings. Right now the favourites list is very limiting. Some of my favourite stickers get bumped out of the list because I ended up using some uncommon sticker and now I need to go and sift through my hundreds/thousands of stickers to find that one again. One could argue that maybe I should just make my own sticker pack with all my favourites, but that seems like a very laborious activity when I just want to be able to pin a few from the sticker packs I already have to the top of my list.

So in summary, I'd like to see a feature added to Telegram where I can create account-local groups to pin stickers from packs I have saved, instead of the current favourites list which is small and volatile.

**Comment by stek29** [comment]
API issue, sorry.

**Comment by karai17** [comment]
Where can I post a feature request on the API? I could not find a place.

**Comment by stek29** [comment]
Twitter or in app support
